spec/services/proofing/aamva/request/verification_request_spec.rb

```diff
@@ -356,4 +356,33 @@
       end
     end
   end
+
+  describe 'compound last names' do
+    let(:applicant_data) { super().merge(last_name: 'McFirst McSecond') }
```

Comment:
would it be preferable to turn the last name into a variable like we do for `state_id_number` and `state_id_jurisdiction`?